pending/20-dancer2-plugin-minion.pod

```diff
@@ -0,0 +1,232 @@
+=head1 Dancer2::Plugin::Minion (aka, Using Minion in Dancer Apps, Revisited)
+
+In 2018, L<I wrote about my experience using Minion|http://advent.perldancer.org/2018/16> at 
+my employer at that time. Since then, I changed to another employer, where again, L<Minion> was
+the appropriate solution to the issue of long-running jobs in our application. They desired
+to have a more Dancer-ish, polished, and integrated solution, and thanks to them,
+L<Dancer2::Plugin::Minion> was born.
+
+I don't see a lot of value rehashing the rationale behind this - I think this was covered
+pretty well in the 2018 article - so feel free to pop back there if you're looking for 
+some backstory and analysis. With that, let's move on to...
+
+=head2 The Code!
+
+We're going to build an image uploader that lets a child uoload a picture of an item on
+their Christmas list to Santa. We're going to save that image to a public image directory,
+and then it will generate a variety of sizes of thumbnails for that image. We don't want
+the client to have to wait on the generation to complete (they have other things to upload
+to Santa after all!), so we will use Minion to generate those thumbnails in the background.
+
+It requires a minimal amount of configuration to set up L<Minion> in your L<Dancer2> apps:
+
+    plugins:
+        Minion:
+            dsn: sqlite:test.db
+            backend: SQLite
+
+Fill in whatever values for DSN and backend match your existing Minion setup. And that's
+it - your L<Dancer2> app can now talk to L<Minion>!
+
+The plugin exports a keyword, C<minion>, that exposes all of L<Minion> to your L<Dancer2> 
+application. I'm not promising it will be as seamless as an experience as the one when
+you are building L<Mojolicious> applications, but it is really powerful.
+
+Two more keywords are created by the plugin, C<add_task> and C<enqueue>, that map directly 
+to the same methods available in C<Minion>. These tasks are common enough to warrant having
+their own keywords to save you the little bit of additional typing to use them.
+
+Lastly, the keyword C<minion_app> creates a Minion application, which you can then mount
+via L<Plack> (if you want to have an admin dashboard for Minion in your Dancer app), or 
+if you want to enable the Minion CLI in your Dancer apps. I will demonstrate this below.
+
+Here's a totally fabricated example to demonstrate how each of these methods can be
```

Comment:
so now that you have the introduction to the example, you don't need this line, 
it should probably just say something like "lets see what that looks like" or so

Reply:
Done!